MessengerOnFileForLinux/DisplayChat/src/NcursesPrintToMainWindowOperationWrapper.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <string>
+
+#include "NcursesPrintOperationWrapper.hpp"
+
+namespace
+{
+static constexpr auto hello = "Hello ";
+static constexpr auto enterPassword = "Enter the password : ";
+static constexpr auto informationToDisplayInMainWindow = "You are currently logged as ";
+static constexpr auto informationToDisplayDuringSignIn = ". If you want use messenger, you need to log in.";
+static constexpr auto informationToDisplayDuringRegistration =
+        ". You are using messenger first time. You need create account. Enter your password twice.";
+
+}//namespace
+
+
+class NcursesPrintToMainWindowOperationWrapper : public NcursesPrintOperationWrapper
```

Comment:
A co sądzisz by interfejs publiczny to była jedna funkcja printInformation, która przyjmuje enuma z jaki rodzaj informacji powinien zostać teraz wydrukowany?

Reply:
Nie podoba mi się ten pomysł. Tak rozbity interfejs jest dużo czytelniejszy w użyciu, a po drugie ta jedna metoda i sprowadzałby się do wielkiego switch/case

Reply:
ok, też tak myślałem, stąd tylko propozycja